gsa/src/gmp/models/filter.js

```diff
@@ -759,6 +759,8 @@ export const DEFAULT_FALLBACK_FILTER = Filter.fromString('sort=name first=1');
 
 export const RESET_FILTER = Filter.fromString('first=1');
 
+export const DEFAULT_ROWS_PER_PAGE = 10;
```

Comment:
As per request of PS, this should be much higher. About 100 rows should fit user requirements better.
```suggestion
export const DEFAULT_ROWS_PER_PAGE = 100;
```

Reply:
As discussed the fallback will be 50 now.